plugin/trino-hive/src/main/java/io/trino/plugin/hive/ViewReaderUtil.java

```diff
@@ -147,7 +147,7 @@ public static boolean isTrinoMaterializedView(Table table)
 
     public static boolean isTrinoMaterializedView(Map<String, String> tableParameters)
     {
-        return isPrestoView(tableParameters) && tableParameters.get(TABLE_COMMENT).equalsIgnoreCase(ICEBERG_MATERIALIZED_VIEW_COMMENT);
+        return isTrinoView(tableParameters) && tableParameters.get(TABLE_COMMENT).equalsIgnoreCase(ICEBERG_MATERIALIZED_VIEW_COMMENT);
```

Comment:
is it obvious that `isTrinoView` can be true also for materialized view?

to me, a materialized view is a separate type, not a subclass of view. especially given that it has separate handling in ConnectorMetadata

that's perhaps why you introduced `isViewOrMaterializedView` method -- you didn't call it `isView`, as would be sufficient if materialized views were always treated as a subcategory of views

Reply:
i am doing this in https://github.com/trinodb/trino/pull/18570